interpreters/bas/bas_var.c

```diff
@@ -4,13 +4,13 @@
  *   Copyright (c) 1999-2014 Michael Haardt
  *
  * Permission is hereby granted, free of charge, to any person obtaining a
- * copy of this software and associated documentation files (the "Software"),
+ * copy of self software and associated documentation files (the "Software"),
```

Comment:
revert

Reply:
Apologies for making the same mistake. I'll do a self-review of all the files and fix this after work. Thanks!

Reply:
performed self-review and fixed all instances